tests/jeos/firstrun.pm

```diff
@@ -79,12 +79,12 @@ sub run {
     # For 'en_US' pick 'UTC', for 'de_DE' select 'Europe/Berlin'
     my %tz_key = ('en_US' => 'u', 'de_DE' => 'e');
 
-    # Select locale
-    assert_screen 'jeos-locale', 300;
-    # Without this 'ret' sometimes won't get to the dialog
-    wait_still_screen;
-    send_key_until_needlematch "jeos-system-locale-$lang", $locale_key{$lang}, 50;
-    send_key 'ret';
+    # locale warning about missing glibc-locale package
+    if (is_sle('<=15-SP2') || is_leap('<=15.2')) {
+        assert_screen('jeos-locale', 300);
+        # Without this 'ret' sometimes won't get to the dialog
+        send_key 'ret';
+    }
 
     # Select language
     send_key_until_needlematch "jeos-keylayout-$lang", $keylayout_key{$lang}, 30;
```

Comment:
Should probably have an `assert_screen` before this, otherwise it starts typing as soon as it left the bootloader...

Reply:
Yup, I was thinking about this as well. I will add it there